bin/brew

```diff
@@ -66,11 +66,12 @@ done
 # test-bot does environment filtering itself
 if [[ -z "$HOMEBREW_NO_ENV_FILTERING" && "$1" != "test-bot" ]]
 then
-  PATH="/usr/bin:/bin:/usr/sbin:/sbin"
+  PATH="/usr/bin:/bin:/usr/sbin:/sbin:/usr/bin/applets"
 
   FILTERED_ENV=()
   # Filter all but the specific variables.
   for VAR in HOME SHELL PATH TERM LOGNAME USER CI CIRCLECI TRAVIS SSH_AUTH_SOCK SUDO_ASKPASS \
+             LD_LIBRARY_PATH ANDROID_ROOT ANDROID_DATA \
```

Comment:
This patch cannot go upstream. As a workaround, you can use `HOMEBREW_NO_ENV_FILTERING=1`

Reply:
Added the environment variable to the android wiki page, and suggested upstream that termux enable RUNPATH by default like the rest of Linux. IMHO Homebrew should as well, but I'll leave well enough alone :)